src/main/java/TechDetectives/HobbyFinder/controllers/HomeController.java

```diff
@@ -0,0 +1,84 @@
+package TechDetectives.HobbyFinder.controllers;
+
+
+import TechDetectives.HobbyFinder.Models.Data.CategoryRepository;
+
+import TechDetectives.HobbyFinder.Models.Category;
+import TechDetectives.HobbyFinder.Models.Data.HobbyRepository;
+import TechDetectives.HobbyFinder.Models.Hobby;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@Controller
+public class HomeController {
```

Comment:
Should this be named HobbyController, since its for the endpoints for hobbies?

Reply:
name changed to HobbyController